subsys/bluetooth/shell/bt.c

```diff
@@ -3739,6 +3739,41 @@ static int cmd_bondable(const struct shell *sh, size_t argc, char *argv[])
 	return 0;
 }
 
+#if defined(CONFIG_BT_BONDABLE_PER_CONNECTION)
+static int cmd_conn_bondable(const struct shell *sh, size_t argc, char *argv[])
+{
+	int res;
+	const char *action;
+	bool enable;
+
+	if (!default_conn) {
+		shell_print(sh, "Not connected");
+		return 0;
+	}
+
+	action = argv[1];
+
+	if (!strcmp(action, "on")) {
+		enable = true;
+	} else if (!strcmp(action, "off")) {
+		enable = false;
+	} else {
+		shell_help(sh);
+		return 0;
+	}
```

Comment:
Suggest to use `shell_strtobool` which handles multiple ways of setting boolean values and has error checking

Reply:
> Or I need to give a initialized value before passing the parameter `err`. Interesting.

Yes, they have been designed to require the `err` being initialized to 0. I don't recall the reason why :)